src/app/components/actions/actions.component.html

```diff
@@ -0,0 +1,95 @@
+<clr-main-container class="main-container">
+    <clr-header class="header header-1">
+        <div class="branding">
+            <a routerLink="/home"  routerLinkActive="active" class="nav-link">
+                <img src="assets/dojo-transparent-white-outline.svg" alt="A dojo" class="mtdj__logo">
+                <span class="title">Math Dojo</span>
+            </a>
+        </div>
+    </clr-header>
+    <nav class="subnav">
+        <ul class="nav">
+            <li class="nav-item">
+                <a class="nav-link" routerLink="/actions">Actions</a>
+            </li>
+    
+        </ul>
+    </nav>
+
+    <div class="content-container" id="primary">
+        <div class="content-area">
+            <div class="clr-row clr-align-items-center site-main">
+                <div class="clr-col-6">
+                    <h1>Stuff admins do</h1>
+                    <div class="clr-row">
+                        <div class="clr-col-4 mtdj__actionspage_display">
+                            <div class="card">
+                                <div class="card-header">
+                                    <h3>View and Approve Questions</h3>
+                                </div>
+                                <div class="card-block">
+                                    <div class="card-text">
+                                        Access the queue of quarantined questions waiting to be approved
+                                    </div>
+                                </div>
+                                <div class="card-footer">
+                                    <a routerLink="/actions/approve" routerLinkActive="active" class="btn btn-primary card clickable mtdj__actionspage_card">Approve Questions</a>
```

Comment:
I don't think you need this `routerLinkActive`. It gives the link here a strange appearance:
![image](https://user-images.githubusercontent.com/20339399/88506840-d0b7a700-cfd2-11ea-9999-9aff125ebee0.png)